stonesoup/sensor/base.py

```diff
@@ -115,7 +117,14 @@ def orientation(self) -> Optional[StateVector]:
             """
         if self.movement_controller is None:
             return None
-        return self.movement_controller.orientation + self.rotation_offset
+        x_axis = (1, 0, 0)
+        rotmat = build_rotation_matrix_xyz(-self.movement_controller.orientation) @ \
+            build_rotation_matrix_xyz(-self.rotation_offset)
```

Comment:
```suggestion
        rotmat = (build_rotation_matrix_xyz(-self.movement_controller.orientation) @ 
            build_rotation_matrix_xyz(-self.rotation_offset))
```
Marginal preference for brackets instead of `\` as a line continuation char